homeassistant/components/atag/climate.py

```diff
@@ -66,9 +66,11 @@ def hvac_action(self) -> Optional[str]:
     @property
     def temperature_unit(self):
         """Return the unit of measurement."""
-        if self.coordinator.atag.climate.temp_unit in [TEMP_CELSIUS, TEMP_FAHRENHEIT]:
-            return self.coordinator.atag.climate.temp_unit
-        return None
+        return (
+            TEMP_CELSIUS
+            if self.coordinator.atag.climate.temp_unit == TEMP_CELSIUS
+            else TEMP_FAHRENHEIT
+        )
 
```

Comment:
We usually break up a ternary into an if block if it become multi line.

Reply:
I reduced it to one line because otherwise an additional test is necessary, which seems superfluous. In fact, it could simply be `return self.coordinator.atag.climate.temp_unit` because that property only returns TEMP_CELSIUS or TEMP_FAHRENHEIT. Not sure whether that is acceptable?

Reply:
I think its fine to simply `return self.coordinator.atag.climate.temp_unit`  It doesn't make it any more risky

Reply:
Cool, will do!